client/state/plugins/installed/selectors.js

```diff
@@ -105,6 +105,16 @@ export function getPlugins( state, siteIds, pluginFilter ) {
 	return sortBy( pluginList, ( item ) => item.slug.toLowerCase() );
 }
 
+export const getPlugins = createSelector(
```

Comment:
This memoizes the `getPlugins` selector. This improves performance, still this selector is expensive because it is being called with many different sets of siteIds.